.azure-pipelines/client.yml

```diff
@@ -28,38 +24,106 @@ jobs:
+        displayName: 'Clean up unsigned build tool allowance'
+        condition: always()
+
+      - task: PublishBuildArtifacts@1
+        displayName: 'Publish Artifacts'
+        inputs:
+          PathtoPublish: 'binaries\packages'
+          ArtifactName: packages
+
+  - job: 'Audit_Windows'
+
+    pool:
+      vmImage: 'vs2017-win2016'
+
+    steps:
+      - script: |
+         "%ProgramFiles(x86)%\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.1 Tools\sn.exe" -Vr *,31bf3856ad364e35
+        displayName: 'Allow unsigned build tool'
 
       - task: MSBuild@1
-        displayName: 'Build NuGet Packages'
+        displayName: 'Init Build'
         inputs:
           solution: build.proj
-          msbuildArguments: '/t:PackageNoBuild /p:Scope=$(source.scope) /p:Configuration=Release /flp:v=diag /flp:Summary;Verbosity=minimal;LogFile=PkgBuild.sum /flp1:warningsonly;logfile=PkgBuild.wrn /flp2:errorsonly;logfile=PkgBuild.err /p:DebugOutput=true /p:DebugTrace=$(system.debug)'
+          msbuildArguments: '/t:Init /flp:LogFile=msbuild.Initlog;Verbosity=normal /p:UseLatest=$(build.useLatest)'
 
-      - script: |
-         set _VSWHERE="%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
-         for /f "usebackq tokens=*" %%i in (`%_VSWHERE% -latest -prerelease -property installationPath`) do set _VSCOMNTOOLS=%%i\Common7\Tools
-         echo "%_VSCOMNTOOLS%\VsDevCmd.bat"
-         call "%_VSCOMNTOOLS%\VsDevCmd.bat"
-         
-         sn -Vl
-         sn -Vu *,31bf3856ad364e35
-        displayName: 'Clean up unsigned build tool allowance'
-        condition: always()
+      - task: MSBuild@1
+        displayName: 'Run Tests'
+        inputs:
+          solution: build.proj
+          msbuildArguments: '/t:RunTests /p:Scope=$(scope) /p:Configuration=Release /clp:ShowTimestamp /flp:Summary;Verbosity=minimal;LogFile=FullTest.sum /flp1:warningsonly;logfile=FullTest.wrn /flp2:errorsonly;logfile=FullTest.err /p:DebugTrace=$(system.debug) /p:CodeSign=false /p:AuthenticodeSign=false /p:RunGlobalReadinessTools=false /p:EnablePreBuildTarget=false /p:EnablePostBuildTarget=false'
 
       - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
         displayName: 'Component Detection'
         # ComponentGovernance is currently unable to run on pull requests of public projects.  Running on 
         # scheduled builds should be good enough.
         condition: and(succeeded(), eq(variables['Build.Reason'], 'Schedule'))
 
-      - task: PublishBuildArtifacts@1
-        displayName: 'Publish Artifacts'
-        inputs:
-          PathtoPublish: 'binaries\packages'
-          ArtifactName: packages
+      - script: |
+         "%ProgramFiles(x86)%\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.1 Tools\sn.exe" -Vu *,31bf3856ad364e35
+        displayName: 'Clean up unsigned build tool allowance'
+        condition: always()
+
+  - job: 'Audit_Linux'
```

Comment:
Lets call these Test_Windows and Test_Linux instead of Audit.

Reply:
Absolutely.